patric_tools/amr.py

```diff
@@ -68,11 +69,13 @@ def get_amr_data_by_species_and_antibiotic(antibiotic, species=None, drop_interm
 
     # Drop strange phenotype names that were encountered
     amr = amr.loc[amr.resistant_phenotype != "Not defined"]
-    amr = amr.loc[amr.resistant_phenotype != "Non-susceptible"]
-    amr = amr.loc[amr.resistant_phenotype != "Nonsusceptible"]
+    # amr = amr.loc[amr.resistant_phenotype != "Non-susceptible"]
+    # amr = amr.loc[amr.resistant_phenotype != "Nonsusceptible"]
+    # amr = amr.loc[amr.resistant_phenotype != "Susceptible-dose dependent"]
+    amr = amr.loc[amr.resistant_phenotype != "S"]
```

Comment:
I think we can consider "S" phenotypes as susceptible. I pulled the latest AMR data, selected a genome with the "S" phenotype and checked the biosample info. It's listed as "susceptible" to the drug in question (https://www.ncbi.nlm.nih.gov/biosample/SRS2577435).